object/check_password_expired.go

```diff
@@ -0,0 +1,26 @@
+package object
+
+import (
+	"time"
+)
+
+func checkPasswordExpired(user *User) bool {
+	organization, err := GetOrganizationByUser(user)
+	if err != nil {
+		return false
+	}
+	passwordExpireDays := organization.PasswordExpireDays
+	if passwordExpireDays <= 0 {
+		return false
+	}
+	lastChangePasswordTime := user.LastChangePasswordTime
+	if lastChangePasswordTime == "" {
+		return false
```

Comment:
If I didn't ever change password, I can bypass this check, is this OK? It's a security hole!

You should use user created time if lastChangePasswordTime is empty

If user created time is also empty, then return error